packages/client/src/pages/projects/NewProject.tsx

```diff
@@ -74,6 +86,11 @@ export const NewProject: React.FC = () => {
 
   return (
     <>
+      {error && (
```

Comment:
With the project name uniqueness error built into the form error handling, this can be removed. Then an error would only take place due to a bug (user not having permission and somehow still accessing this page, backend issue, etc)